dependencies.list

```diff
@@ -1,3 +1,5 @@
 # Version of MongoDB Realm used by integration tests
 # See https://github.com/realm/ci/packages/147854 for available versions
 MONGODB_REALM_SERVER=2022-05-21
+REALM_BAAS_GIT_HASH=87e4edf99a6510aa8468450e87af69dcfc74abe7
```

Comment:
Could we add easy-to-access links to their repos, similar to the link to packages?